lib/generators/rolemodel/mailers/templates/app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class UserMailer < ApplicationMailer
+  default from: 'notifications@example.com'
+
+  def welcome_email
```

Comment:
What are we thinking this Mailer would be used for? Is this kind of a hello world example?

Reply:
Yes, it's just a sample. I should change the name since devise has its own user mailer things.